cf-backup/README.md

```diff
@@ -0,0 +1,27 @@
+# Cloudfiles backup image
+
+This image will upload (backup) a directory to cloudfiles every minute using cron and rack.
```

Comment:
Cloud Files instead of cloudfiles

rack should link to https://developer.rackspace.com/docs/rack-cli/

This image would be much more generally useful if you could configure it to backup every x minutes/hours/days. It's probably worthwhile to even just let the user set the cron schedule to whatever they like.